AdobeStockImage/Test/Unit/Model/SaveImageTest.php

```diff
@@ -180,9 +181,9 @@ public function assetProvider(): array
      * Get document
      *
      * @param string|null $path
-     * @return Document|MockObject
+     * @return MockObject
```

Comment:
I think `Document|MockObject` is correct param type here

Reply:
PHPStan validation is failing here. 
Basically return type hint should be the same as @return tag in DocBlock.
So I had to choose between MockObject and Document. I decided to use MockObject because MockObject's interface is used after getting `getDocument` result.